rllib/tuned_examples/dqn/cartpole-r2d2.yaml

```diff
@@ -0,0 +1,30 @@
+cartpole-r2d2:
+    env: CartPole-v0
+    run: R2D2
+    stop:
+        episode_reward_mean: 150
+        timesteps_total: 1000000
+    config:
+        # Works for both torch and tf.
+        framework: tf
+
```

Comment:
Small nitpick remove the `\n` to be consistent with all yaml files.

Reply:
done